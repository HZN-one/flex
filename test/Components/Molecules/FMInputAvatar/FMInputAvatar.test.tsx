/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";
import { FMInputAvatar } from "@Molecules";

import { render } from "@testing-library/react";

describe("FMCard", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FMInputAvatar
        testID="input-avatar-test"
        infoText="minimum image size 100 x 100 (max. 200 Kb). extention file must be in .PNG"
        buttonLabel="Upload Company Logo"
        acceptInput="image/*"
      />
    );
    expect(container).toMatchSnapshot();
  });
});
