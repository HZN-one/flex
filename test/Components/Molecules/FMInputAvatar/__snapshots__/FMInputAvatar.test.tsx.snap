// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMCard should match snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-r66v1n"
    data-testid="input-avatar-test"
  >
    <div
      class="MuiAvatar-root MuiAvatar-circular css-1yctjaw-MuiAvatar-root"
      data-testid="avatar-company-logo"
    >
      <img
        class="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
        src="https://storage.googleapis.com/dev-hzn-assets/icons/inputAvatar.png"
      />
    </div>
    <div
      class="css-1tg5evk-MuiStack-root"
    >
      <span
        aria-labelledby="upload-company-logo-id"
        class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
        data-testid="typography-upload-company-logo"
      >
        minimum image size 100 x 100 (max. 200 Kb). extention file must be in .PNG
      </span>
      <label>
        <input
          accept="image/*"
          hidden=""
          type="file"
        />
        <span
          class="MuiButton-root MuiButton-outlined MuiButton-outlinedSecondary MuiButton-sizeSmall MuiButton-outlinedSizeSmall MuiButtonBase-root  css-1kc3inn-MuiButtonBase-root-MuiButton-root"
          role="button"
          tabindex="0"
        >
          Upload Company Logo
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </span>
      </label>
    </div>
  </div>
</div>
`;
